.editorconfig

```diff
@@ -0,0 +1,74 @@
+# Visual Studio generated .editorconfig file with C++ settings.
+root = true
+
+[*.{c++,cc,cpp,cppm,cxx,h,h++,hh,hpp,hxx,inl,ipp,ixx,tlh,tli}]
+
+indent_style = space
+indent_size = 4
+insert_final_newline = true
+charset = utf-8
+
+# Visual C++ Code Style settings
+
+cpp_generate_documentation_comments = doxygen_triple_slash
+
+# Visual C++ Formatting settings
+
+cpp_indent_braces = false
+cpp_indent_multi_line_relative_to = statement_begin
+cpp_indent_within_parentheses = indent
+cpp_indent_preserve_within_parentheses = true
+cpp_indent_case_contents = true
+cpp_indent_case_labels = false
+cpp_indent_case_contents_when_block = false
+cpp_indent_lambda_braces_when_parameter = true
+cpp_indent_goto_labels = one_left
+cpp_indent_preprocessor = leftmost_column
+cpp_indent_access_specifiers = false
+cpp_indent_namespace_contents = true
+cpp_indent_preserve_comments = false
+cpp_new_line_before_open_brace_namespace = ignore
+cpp_new_line_before_open_brace_type = ignore
+cpp_new_line_before_open_brace_function = ignore
+cpp_new_line_before_open_brace_block = ignore
+cpp_new_line_before_open_brace_lambda = ignore
+cpp_new_line_scope_braces_on_separate_lines = true
```

Comment:
Did you change this from the default one? If so, why? When I auto-generated the file on my machine it is put to:
`cpp_new_line_scope_braces_on_separate_lines = false`

Reply:
Yes, this is changed (according to the convention used so far in the codebase and according to my prerferences).
I prefer things this way
![image](https://user-images.githubusercontent.com/4881321/216312613-b29f729a-a76e-4874-a2c5-15151b6596a7.png)
and **not** like this
![image](https://user-images.githubusercontent.com/4881321/216312689-c2a1dfd3-9ed2-4162-84eb-f7923d5ed6e2.png)